docs/tasks/stateful-sets/deleting-a-statefulset.md

```diff
@@ -0,0 +1,102 @@
+---
+assignees:
+- bprashanth
+- erictune
+- foxish
+- janetkuo
+- smarterclayton
+
+---
+
+{% capture overview %}
+This task shows you how to delete a Stateful Set.
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* If you are not using Stateful Sets, you can skip this task now. 
+
+{% endcapture %}
+
+{% capture steps %}
+
+### Deleting a StatefulSet
+
+A Stateful Set can be deleted like other resources in kubernetes. 
+
+```shell
+$ kubectl delete -f statefulset.yaml 
+statefulset "web" deleted
+
+```
+
+
+```shell
+$ kubectl delete statefulsets web
+statefulset "web" deleted
+
+```
+
+The associated headless service may need to be deleted separately after the Stateful Set itself is deleted.
+
+```shell
+$ kubectl delete service web
+service "web" deleted
+
+```
+
+Deleting a Stateful Set through kubectl will scale it down to 0, thereby deleting all pods that are a part of it. If you wish to delete just the Stateful Set and not the pods, please use --cascade=false.
```

Comment:
According to the [style guide](http://kubernetes.io/docs/contribute/style-guide/#use-simple-and-direct-language), we shouldn't say "please".